google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/RemoveFeedItemAttributeValue.java

```diff
@@ -0,0 +1,257 @@
+ * FLIGHT_SALE_PRICE attribute must be present on the feed.
+ */
+public class RemoveFeedItemAttributeValue {
+  private static final int PAGE_SIZE = 1_000;
+
+  private static class RemoveFeedItemAttributeValueParams extends CodeSampleParams {
+
+    @Parameter(names = ArgumentNames.CUSTOMER_ID, required = true)
+    private long customerId;
+
+    @Parameter(names = ArgumentNames.FEED_ID, required = true)
+    private long feedId;
+
+    @Parameter(names = ArgumentNames.FEED_ITEM_ID, required = true)
+    private long feedItemId;
+  }
+
+  public static void main(String[] args) {
+    RemoveFeedItemAttributeValueParams params = new RemoveFeedItemAttributeValueParams();
+    if (!params.parseArguments(args)) {
+
+      // Either pass the required parameters for this example on the command line, or insert them
+      // into the code here. See the parameter class definition above for descriptions.
+      params.customerId = Long.parseLong("INSERT_CUSTOMER_ID_HERE");
+      params.feedId = Long.parseLong("INSERT_FEED_ID_HERE");
+      params.feedItemId = Long.parseLong("INSERT_FEED_ITEM_ID_HERE");
+    }
+
+    GoogleAdsClient googleAdsClient;
+    try {
+      googleAdsClient = GoogleAdsClient.newBuilder().fromPropertiesFile().build();
+    } catch (FileNotFoundException fnfe) {
+      System.err.printf(
+          "Failed to load GoogleAdsClient configuration from file. Exception: %s%n", fnfe);
+      return;
+    } catch (IOException ioe) {
+      System.err.printf("Failed to create GoogleAdsClient. Exception: %s%n", ioe);
+      return;
+    }
+
+    try {
+      new RemoveFeedItemAttributeValue()
+          .runExample(googleAdsClient, params.customerId, params.feedId, params.feedItemId);
+    } catch (GoogleAdsException gae) {
+      // GoogleAdsException is the base class for most exceptions thrown by an API request.
+      // Instances of this exception have a message and a GoogleAdsFailure that contains a
+      // collection of GoogleAdsErrors that indicate the underlying causes of the
+      // GoogleAdsException.
+      System.err.printf(
+          "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+          gae.getRequestId());
+      int i = 0;
+      for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()) {
+        System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+      }
+    }
+  }
+
+  /**
+   * Runs the example.
+   *
+   * @param googleAdsClient the Google Ads API client.
+   * @param customerId the client customer ID in which to create criterion.
+   * @param feedId the ID of the feed on which the feedItemAttribute value will be removed.
+   * @param feedItemId the ID of the feed item on which the feedItemAttribute value will be removed.
+   * @throws GoogleAdsException if an API request failed with one or more service errors.
+   */
+  private void runExample(
+      GoogleAdsClient googleAdsClient, long customerId, long feedId, long feedItemId) {
```

Comment:
Replace the params with the RemoveFeedItemAttributeValueParams class (same comment re. run all examples).